pact-jvm-consumer/src/main/java/au/com/dius/pact/consumer/dsl/PactDslResponse.java

```diff
@@ -21,18 +19,18 @@
 import scala.collection.JavaConversions$;
 
 import javax.xml.transform.TransformerException;
-import java.util.Arrays;
-import java.util.Collections;
 import java.util.HashMap;
 import java.util.Map;
 import java.util.function.Supplier;
 
 public class PactDslResponse {
     private static final String CONTENT_TYPE = "Content-Type";
+    static final String DEFAULT_JSON_CONTENT_TYPE_REGEX = "application/json;\\s?charset=(utf|UTF)-8";
```

Comment:
I think the regex should accept all charsets, not just UTF-8. On Windows it could be one of the Windows specific charsets.